src/app/shared/components/filters-list/city-filter/city-filter.component.scss

```diff
@@ -10,14 +10,17 @@
   padding: 0.2rem;
   width: auto;
 }
+
 .input-filter {
   text-transform: capitalize;
   font-family: 'Innerspace';
   color: #000000;
   font-size: 1.125rem;
   line-height: 1.25rem;
   font-weight: 500;
+  min-width: 290px;
```

Comment:
It now looks good on any devices.
This selector wont affect any potential new mat-autocomplete because it have id selector which unique and due to scope of css component files

Reply:
cdk-overlay-id can be different, so don't use it.
You can provide class as input for autocomplete or use panelWidth option.
Ref: https://v14.material.angular.io/components/autocomplete/api

Reply:
Is it okay [panelWidth]="300"?

Reply:
Yes, its fine